api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/OperationalStep.java

```diff
@@ -56,17 +57,27 @@ public OperationResult run(OperationData data) {
                 return result;
             }
         }
-        if (nextStep != null) {
+        if (!NullStep.getInstance().equals(nextStep)) {
```

Comment:
Why do we have run method in NullStep if it is not invoked?

Reply:
Fixed